test/Posts/PostsTest.php

```diff
@@ -43,11 +43,12 @@ public function testLookup() {
     * @expectedException Traackr\MissingParameterException
     */
    public function testSearchInvalidParameter() {
```

Comment:
Should update the name of the test here. The parameter is no longer invalid. Are we just testing a successful search with no posts?